### DISCUSSION TITLE
min_area filter not working??

### DISCUSSION THREAD

**Discussion by bm55** [blakeblackshear/frigate#3323] [Ask A Question]
I have a zone called "frontDoorA" which has person filter with min_area = 6200. Shouldn't that prevent it from activating the zone when it detects person's area smaller than 6200?
![image](https://user-images.githubusercontent.com/7528001/172929818-cdf3a80e-d748-4d28-855c-43547920af8b.png)
![image](https://user-images.githubusercontent.com/7528001/172929870-3600d233-366a-4152-a4ca-77339bbba24f.png)

  back:
    ffmpeg:
      inputs:
        - path: http://192.168.0.138/flv?port=1935&app=bcs&stream=channel0_ext.bcs&user=admin&password={FRIGATE_CAM_PASSWORD}
        #- path : rtmp://192.168.0.138:1935/bcs/channel0_ext.bcs?channel=0&stream=0&user=admin&password={FRIGATE_CAM_PASSWORD}   
          roles:
            - detect
            - rtmp 
        - path: http://192.168.0.138/flv?port=1935&app=bcs&stream=channel0_main.bcs&user=admin&password={FRIGATE_CAM_PASSWORD}
        #- path : rtmp://192.168.0.138:1935/bcs/channel0_main.bcs?channel=0&stream=0&user=admin&password={FRIGATE_CAM_PASSWORD}   
          roles:
            - record

    detect:
      width: 640
      height: 480 
    motion:
      threshold: 20 #25 motion sensitive
      contour_area: 30 #30 small object
      delta_alpha: 0.4 #0.2 fast moving object 
      mask:
        - 352,281,396,146,331,0,256,0,298,196,258,326
        - 640,120,640,47,592,85,609,163,640,190  
        - 557,110,585,114,588,73,564,55,524,56
        - 30,283,51,258,95,313,64,341 
        - 551,436,518,459,565,459,607,459,575,246,501,270
    objects:
      track:
        - person
        - car
        - dog
        - cat
        - bicycle
        - bus
      #filters:
        #person:
          #min_score: 0.5
          #threshold: 0.6
          #min_area: 800
          #mask:
            #- 496,480,583,480,593,449,493,449 # prevent person object from detect (Avacado plant)
    zones:
      frontDoorA:
        coordinates: 640,480,640,210,619,170,371,243,387,326,209,400,132,269,0,274,0,480
        filters:
          person:
            min_score: 0.5
            threshold: 0.7
            min_area: 6200
      frontDoorB:
        coordinates: 365,32,500,21,601,121,640,255,373,279
        filters:
          person:
            min_score: 0.5
            threshold: 0.7
      backRoad:
        coordinates: 599,121,544,54,484,31,348,28,349,0,640,0,640,143
    mqtt:
      crop: true
      quality: 100 
      timestamp: true
      height: 480

**Comment by NickM-27** [comment]
Once an object is part of a zone it remains part of the zone, that filter only applies at the beginning. 

So odds are an actual person walked by, activated the zone, then walked inside and this false detection occurred thinking it was the same "person" that never left the frame

**Reply by bm55** [reply]
@NickM-27 That is likely what happened here - I do see a real person at the beginning of this long video clip. I will re-start frigate, need a better model that doesn't detect shoes as person :)
